packages/react/src/patterns/sub-patterns/PictogramItem/__stories__/PictogramItem.stories.js

```diff
@@ -0,0 +1,66 @@
+import './index.scss';
+
+import { Desktop, Pattern, Touch } from '@carbon/pictograms-react';
+import { text, withKnobs, select } from '@storybook/addon-knobs';
+import PictogramItem from '../PictogramItem';
+import React from 'react';
+// import readme from '../README.md';
+import { storiesOf } from '@storybook/react';
+
+storiesOf('Patterns (Sub-Patterns)|PictogramItem', module)
+  .addDecorator(withKnobs)
+  //   .addParameters({
+  //     readme: {
+  //       sidebar: readme,
+  //     },
+  //   })
```

Comment:
README should be turned back on

Reply:
@jeffchew I've turned it back on, thanks!